### ISSUE TITLE
Will there be documentation on creating a fully custom theme for Nextra?

### ISSUE THREAD

**Issue by kyrylolvov** [shuding/nextra#2861]

**Comment by KhoiUna** [comment]
You can check this out: https://nextra.site/docs/custom-theme

**Comment by kyrylolvov** [comment]
@KhoiUna Yeah, but advanced usage is still in progress. I just cloned Nextra and took a look on how some stuff is being implemented, so all good. Thanks! :)

<img width="752" alt="Screenshot 2024-05-02 at 3 24 36 PM" src="https://github.com/shuding/nextra/assets/70720713/6bc34b90-46a6-4fdf-8633-8682f2427fa2">

**Comment by dimaMachina** [comment]
docs for custom theme was updated https://nextra.site/docs/custom-theme
